lib/ace/mode/ezhil.js

```diff
@@ -0,0 +1,106 @@
+/*
+(C) 2013 Ezhil Language Project
+(C) 2013 Sridhar Ganesan
+*/
```

Comment:
To merge this we need a signed [CLA](https://github.com/ajaxorg/ace/blob/master/CONTRIBUTING.md), and BSD licence similar to [other modes](https://github.com/ajaxorg/ace/blob/master/lib/ace/mode/javascript.js#L1-L29).